samples/peripheral/radio_test/src/radio_test.c

```diff
@@ -13,6 +13,15 @@
 #include <nrfx_timer.h>
 #include <zephyr.h>
 
+#if CONFIG_NRF21540_FEM
+#include "nrf21540.h"
+#endif
+
+/* Radio fast ramp up time in us. */
+#define FAST_RAMP_UP_TIME 40
+/* Radio normal ramp up time in us */
+#define RAMP_UP_TIME 130
+
```

Comment:
This is rather simplistic. It depends on the PHY used, and can vary between SOCs. How important is this precision here?

Reply:
I checked mostly all our SoC PS. It seems like these values are common for all SoC. We do not need much accuracy here, it is enough that these values will not be greater than in reality. They are used to calculate a default delay between pull up FEM RX/TX pin and Radio Start task to ensure that PA/LNA is not enabled too early. User can change this delay by the Vendor-specific command.